great-idea/index.html

```diff
@@ -8,16 +8,79 @@
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
-
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
 </head>
 
   <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
+    <div class='container'>
+      <nav>
+          <a href='services.html'>Services</a>
+          <a href='#'>Product</a>
+          <a href='#'>Vision</a>
+          <a href='#'>Features</a>
+          <a href='#'>About</a>
+          <a href='#'>Contact</a>
+          <img class='logo' src='img/logo.png'>
+      </nav>
+      
+      <header>
+        <div class='innovation'>
+          <h2>Innovation<br> on<br> Demand</h2>
+          <button>Get Started</button>
+        </div>
+        <img src='img/header-img.png'>
+      </header>
+      
+      <main>
+        <section class='top-main'>
+          
+          <div class='features'>
+            <h2>Features</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+          
+          <div class='about'>
+            <h2>About</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+          
+        </section>
+        
+        <img class='mid-img' src='img/mid-page-accent.jpg'>
+        
+        <section class='bottom-main'>
+          <div class='services'>
+            <h2>Services</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+          
+          <div class='product'>
+            <h2>Product</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+          
+          <div class='vision'>
+            <h2>Vision</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+        </section>
+      </main>
+      
+      <address>
```

Comment:
Great use of the address tag, but you actually don't want to put h2 tags inside of address tags. I would wrap the address tag with a section tag, and move the h2 to be a child of the section tag and a sibling of the address tag

Reply:
You can read more about the address tag here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address
notice how it says "no heading content" under the "permitted content" section